src/google/Timeline.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "../common/HTMLWidget", "goog!visualization,1,packages:[timeline]"], factory);
+    } else {
+        root.Timeline = factory(root.d3, root.common_HTMLWidget);
+    }
+}(this, function (d3, HTMLWidget) {
+
+    function Timeline() {
+        HTMLWidget.call(this);
+        
+        this._class = "google_Timeline";
+        this._tag = "div";
+        this._chartType = "Timeline";
+        this._data_google = [];
+
+        this.columns([]);
+        this.data([]);
+    }
+    Timeline.prototype = Object.create(HTMLWidget.prototype);
+
+    /**
+     * Publish Params Common To Other Libraries
+     */
+
+    Timeline.prototype.publish("tooltipIsHtml", "true", "boolean", "Set to false to use SVG-rendered (rather than HTML-rendered) tooltips. See Customizing Tooltip Content for more details.",null,{tags:['Advanced']});
+    
+    Timeline.prototype.publish("tooltipTrigger", "focus", "set", "The user interaction that causes the tooltip to be displayed: 'focus' - The tooltip will be displayed when the user hovers over the element; 'none' - The tooltip will not be displayed.",['none', 'focus'],{tags:['Basic']});
+    
```

Comment:
For consistency no blank line between each publish (I tend to use the blank line to group different properties as appropriate.